wagtail/wagtailcore/blocks/field_block.py

```diff
@@ -581,20 +581,42 @@ class Meta:
 
 class PageChooserBlock(ChooserBlock):
 
-    def __init__(self, target_model='wagtailcore.Page', can_choose_root=False,
+    # TODO: rename target_model to page_type
+    def __init__(self, target_model=None, can_choose_root=False,
                  **kwargs):
-        self._target_model = target_model
+        if target_model:
+            # Convert single string/model into a list
+            if not isinstance(target_model, (list, tuple)):
+                target_model = [target_model]
+        else:
+            target_model = []
+
+        self._target_models = target_model
         self.can_choose_root = can_choose_root
         super(PageChooserBlock, self).__init__(**kwargs)
 
     @cached_property
     def target_model(self):
```

Comment:
As far as I'm aware, the `target_model` property is undocumented and only exists for the use of `PageChooserBlock` itself, so I think it would be appropriate to remove it. (If any external code _is_ accessing the property, it's probably better to let that code fail, to alert the developer that they need to update it to handle multiple page types, rather than returning a not-so-accurate result.)